packages/next/src/export/index.ts

```diff
@@ -547,9 +547,10 @@ export default async function exportApp(
 
     const filteredPaths = exportPaths.filter(
       // Remove API routes
-      (route) =>
-        (exportPathMap[route] as any)._isAppDir ||
-        !isAPIRoute(exportPathMap[route].page)
+      (route) => {
+        const { _isAppDir, page } = exportPathMap[route] || {}
```

Comment:
Can we narrow in on the case this is empty? It shouldn't be as that seems to be leading to a more lower down issue.

Reply:
Agreed but we don't have repro steps so its unclear at which point the entry is missing 

Maybe @ovflowd can create a branch with the failing code again.

Reply:
Is there an easy way I can test this commit of Next.js locally?